transformation_test/testdata/logging_processor-elasticsearch-json/output_fluentbit.yaml

```diff
@@ -0,0 +1,201 @@
+- entries:
```

Comment:
Note: This is a general comment that i mentioned also in https://github.com/GoogleCloudPlatform/ops-agent/pull/1978#discussion_r2214743140. Anotheer way of verifying that the refactor is correct is that the `output_fluentbit.yaml` should be the same before and after the update to `elasticsearch.go`.